src/components/cta/cta.css

```diff
@@ -0,0 +1,253 @@
+h1,
```

Comment:
I think we don't typically include this file. It gets generated or is not used. Either way, it will probably change as we start to add custom elements.